action_type.cpp

```diff
@@ -9,17 +9,16 @@ void test_action_type()
   {
     assert(to_str(action_type::turn_left) == "turn_left");
     assert(to_str(action_type::turn_right) == "turn_right");
-    assert(to_str(action_type::accelerate) == "accelerate");
-    assert(to_str(action_type::brake) == "brake");
-    assert(to_str(action_type::acc_backward) == "acc_backward");
+    assert(to_str(action_type::accelerate_forward) == "accelerate_forward");
+    assert(to_str(action_type::accelerate_backward) == "accelerate_backward");

```

Comment:
@richelbilderbeek given the current parameter set there is no need for `brake`. But when we use some slightly different parameter sets things could be much different. I won't disagree if the team's decision is to put brake back in, and I believe that the new movement system would not conflict with `brake`, just add two new parameters to decide how fast a player can brake in both directions. 

IMO the thing that really matters is how we set those parameters, and this may result in completely different tactile sensations of manipulating. But of course, this is still really far from being seriously considered.

In fact I see a draft pull request as a prototype that is still in progress and prone to change, so I am glad to hear the opinions from you and the entire team. I think that real code may inspire people more, that is why I opened this draft. I really like to make my idea come true ASAP, but I like more to make the implementation even better. So, just like before, please comment if you have any questions, and if there is a need to vote please do so😀

Reply:
However if the new system is adopted, we need to find a new key and map it to `brake`, is it the preferable way?

Reply:
Adding an action, or a key mapped to an action, back is trivial, just go ahead on with your works of wonder :+1: